File: src/agents_config.py
#!/usr/bin/env python3


from src.types_def import AgentConfig

AGENTS_CONFIG: dict[str, AgentConfig] = {
    "opencode": {
        "label": "Opencode",
        "assets": [
            {
                "source": "agents/opencode/opencode.jsonc",
                "target": "opencode.json",
                "type": "file",
            },
            {
                "source": "agents/rules/AGENTS.md",
                "target": "AGENTS.md",
                "type": "file",
            },
        ],
    },
}

File: src/agents-config.ts
import type { AgentsConfigMap } from "./types-def";

export const AGENTS_CONFIG: AgentsConfigMap = {
  opencode: {
    label: "OpenCode",
    assets: [
      {
        source: "agents/opencode/opencode.jsonc",
        target: "opencode.jsonc",
        type: "file",
      },
      {
        source: "agents/rules/AGENTS.md",
        target: "AGENTS.md",
        type: "file",
      },
      {
        source: "agents/opencode/command",
        target: "command",
        type: "dir",
      },
    ],
  },
};
